packages/nimble-components/src/table-column/base/index.ts

```diff
@@ -0,0 +1,31 @@
+import type { ElementStyles, ViewTemplate } from '@microsoft/fast-element';
+import { FoundationElement } from '@microsoft/fast-foundation';
+import type { TableCellState, TableRecord } from '../../table/types';
+
+/**
+ * The base class for table columns
+ */
+export abstract class TableColumn<
+    TCellData extends TableRecord = TableRecord,
+    TColumnConfig = void
+> extends FoundationElement {
+    // The template to use to render the cell content for the column
```

Comment:
Could you use JSDoc formatted comments? `/**`

Reply:
Done.